modules/margin/src/test/java/com/opengamma/sdk/margin/MarginClientTest.java

```diff
@@ -110,7 +123,7 @@ public void test_listCcps_fail() throws Exception {
     ServiceInvoker invoker = ServiceInvoker.of(CREDENTIALS, server.url("/"), new TestingAuthClient());
     MarginClient client = MarginClient.of(invoker);
 
-    assertThrows(IllegalStateException.class, () -> client.listCcps());
+    assertThrows(IllegalStateException.class, client::listCcps);
```

Comment:
Despite what IntelliJ says, this is often not a good refactoring. The method reference can be harder to decode, and is definitely harder to refactor when an additional parameter gets added to the method.